Backend/app/user/user_router.py

```diff
@@ -117,3 +117,42 @@ async def generate_device_UID(user_email: str):
     result = user_service.generate_new_uid(user_email)
 
     return result
+
+
+# ------------------------------------------------------------------------------
+@router.get("/librarian_consult_user")
+def consult_user_by_email(user_email: str):
```

Comment:
All of these librarian endpoints should take an access token and verify it belongs to a librarian.